### DISCUSSION TITLE
Same issue: Docker Container:Admin Password Not Working #819

### DISCUSSION THREAD

**Discussion by jsbah** [DependencyTrack/dependency-track#1181] [Q&A]
Hello all and thanks for helping.

I am following the documentation,
I simply just used curl command to download the docker-compose file and then did 
#docker-compose up -d
I can see the DT login page at http://localhost.org:8080 but using username admin and password admin does not work or even return anything. Clicking the Login button, nothing happens at all.

So I check frontend console logs and see this:
![image](https://user-images.githubusercontent.com/84727224/131404376-18071185-549d-46ed-b72e-0e5227609b96.png)
Thanks for you help

**Comment by nscuro** [comment]
Please check if some other service is running on port 8081 already.  If you publish a port in Docker for Windows, it won‘t show errors if that port is already in use on the host.

Try running this on your host in a powershell Window, maybe you get something back:

```powershell
Get-Process -Id (Get-NetTCPConnection -LocalPort 8081).OwningProcess
```

**Reply by jsbah** [reply]
Thanks @nscuro for this response.
I am using a mac and I have checked if any service is running on port 8081 with: netstat -vanp tcp | grep 8081
and nothing is running on that port. I have also stopped all other containers except for DT. Still nothing.

**Reply by jsbah** [reply]
Here is the docker-compose configuration to meet and surpass minimum requirement for DT
![image](https://user-images.githubusercontent.com/84727224/131527364-549ded1e-8da1-4f37-a9db-3335fe7f665a.png)

and below is docker ps command show that only DT is running and that API is listening on port 8081.
When [Login] button is clicked, the call does not even get to the backend because I follow-monitor the API container logs and no entry upon clicking the [Login] button. Not even anything at that point.
![image](https://user-images.githubusercontent.com/84727224/131528109-af309270-62dd-4e2f-9e00-bd3ff3782dc3.png)

**Reply by jsbah** [reply]
Trying to access the api server via browser:
![image](https://user-images.githubusercontent.com/84727224/131539052-9950b683-42ba-43c2-8f07-b579b5936771.png)

Notice that it looks like the api container is constantly restarting (both backend and fronted started at the same time) :
![image](https://user-images.githubusercontent.com/84727224/131539148-c189df34-6ef6-43a1-bb30-3906bd29134e.png)

**Reply by jsbah** [reply]
Here is an error found in the API container. Not sure whether or not this could be the reason the container is constantly restarting:
![image](https://user-images.githubusercontent.com/84727224/131546816-9ccfbee0-8910-42ef-a113-43c45af0af94.png)

**Reply by jsbah** [reply]
Thanks for all your responses. Many thanks to @stevespringett  for pointing out host resource limit increase. The problem with the restart of the API server is now solved. However, the API server's status goes to "unhealthy" after 5 mins.
![image](https://user-images.githubusercontent.com/84727224/131689679-e27d9f0c-ace6-43da-8aed-4df64aeab49a.png)
Any suggestions on what I can do to troubleshoot why this is happening?

**Reply by jsbah** [reply]
Hello all, thanks for your response again. I have now disabled CORS policy to make sure that is not the problem. However, I am still unable to log into DT. Here is the screenshot of the console log, Please help
![image](https://user-images.githubusercontent.com/84727224/131735545-c1154114-c58f-4182-89e8-807f2b792837.png)

**Reply by jsbah** [reply]
Thanks again guys. Attached is a copy of the API log.
[installation_output.txt](https://github.com/DependencyTrack/dependency-track/files/7094512/installation_output.txt)

**Reply by jsbah** [reply]
Many thanks to @nscur0 and @SteveSpringett, I have finally been able to login.
Here is what I changed to fix the issue:
1. As per @stevespringett , docker *host* on mac will only allocate max 2G to container, I changed this to 7G
2. I used safari's developer tab to disable cross-origin-resource
3. The command "docker-compose down --volume" and "docker-compose up -d" resolved the final error
Thank you all
